Formula/go.rb

```diff
@@ -3,9 +3,9 @@ class Go < Formula
   homepage "https://golang.org"
 
   stable do
-    url "https://storage.googleapis.com/golang/go1.9.1.src.tar.gz"
-    mirror "https://fossies.org/linux/misc/go1.9.1.src.tar.gz"
-    sha256 "a84afc9dc7d64fe0fa84d4d735e2ece23831a22117b50dafc75c1484f1cb550e"
+    url "https://storage.googleapis.com/golang/go1.9.2.src.tar.gz"
+    version "1.9.2"
```

Comment:
It shouldn't be necessary to re-add this line?

Reply:
That's cool if not! I just looked in my shell history for the last `brew bump-formula-pr` that I performed, and it had `--version=`, so I used that. I'll remove this line if it's not needed.

Reply:
It should be okay now. _Should_. Version detection on the `brew` side of things was improved.

Reply:
Swee, removed. I added another commit, can't remember if amending existing commits is cool or not.

Reply:
Everything gets squashed down to one commit per formula changed prior to merge, generally. Sometimes Jenkins can throw a tantrum about force pushes but yeah that's the standard workflow.